### ISSUE TITLE
Firefox - 'no' incorrectly expanded to number when it comes before '[n of m]'

### ISSUE THREAD

**Issue by 36degrees** [nvaccess/nvda#8255]
### Steps to reproduce:
1. Open [this codepen](https://codepen.io/36degrees/pen/PeRNPM) in Firefox (tested in 52.8.0) which includes this code:

```html
<form>
  <fieldset>
    <legend>Have you changed your name?</legend>
    
    <span id="hint">Choose yes or no</span>

    <label>
    <input type="radio" name="changed-name" value="yes" aria-describedby="hint">Yes
    </label>
    
    <label>
    <input type="radio" name="changed-name" value="no" aria-describedby="hint">No
    </label>
  </fieldset>
</form>
```

2. Tab to 'yes' radio button and listen to what NVDA announces

### Expected behavior:
NVDA should announce "Have you changed your name? Grouping, yes radio button not checked, choose yes or **no**, 1 of 2."

### Actual behavior:
NVDA announces "Have you changed your name? Grouping, yes radio button not checked, choose yes or **number** 1 of 2."

### System configuration:
NVDA version:
2017.4

NVDA Installed or portable:
Installed

Other information:
N/A

Windows version:
Windows 10 Version 1709 Build 16299.431

Name and version of other software in use when reproducing the issue:
Firefox 52.8.0

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No

**Comment by Brian1Gaff** [comment]
Aha!
 This would explain some issues I've been having with some forms that seem 
not to respond  very well in checkboxes and radio buttons when there are a 
lot of them it can get very confusing when some work as expected and some do 
not, either needing a page refers or a lot of changing of mode and cursoring 
about.
  Unfortunately the last survey I did like this has now expired but it took 
nearly two hours instead of 20 minutes!
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by ehollig** [comment]
This works as expected for me using NVDA alpha-16157,526d6fca. When I tab to the yes radio button, NVDA announces "Yes  radio button  not checked  Choose yes or no  1 of 2". @36degrees is this still an issue?

**Comment by 36degrees** [comment]
I've just tested this again and it does appears to be fixed in NVDA 2018 with Firefox 60.2.2esr (at least as of 2018.2, which is what I was able to test in)

I now hear "Have you changed your name? Grouping, yes radio button not checked, choose yes or no, 1 of 2." which is what I would expect.

(I also tested again in NVDA 2017.4, with the same version of Firefox, and it still seems to be an issue there, so I think the change that fixed this was in NVDA rather than Firefox)

Thanks!
